app/views/samples/_attribute_values.html.erb

```diff
@@ -4,3 +4,9 @@
       <%= display_attribute(sample, attribute, link: true) %>
     </p>
 <% end %>
+
+<script>
+    $j(document).ready(function() {
+        $j('.term-label').on('click', SampleTypes.expandIRI);
+    });
+</script>
```

Comment:
in the interest of trying to remove inline javascript, this block could be moved to sample_types.js . I don't see `term-label` used anywhere else, but to be more specific I'd use `label.term-label` as the selector.